api/v1beta1/awscluster_types.go

```diff
@@ -186,6 +186,7 @@ type AWSLoadBalancerSpec struct {
 
 	// HealthCheckProtocol sets the protocol type for classic ELB health check target
 	// default value is ClassicELBProtocolSSL
+	// +kubebuilder:validation:Enum=TCP;SSL;HTTP;HTTPS
```

Comment:
We generally dont backport changes to previous api. So changes for v1beta2 would suffice. You could revert this change.

Reply:
sounds good. thank you

Reply:
I have removed it. Thank you.